app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
```

Comment:
Here you can assign default filename. It will change if there is "-f" flag in argv or stay default if not.